lib/elixir/lib/enum.ex

```diff
@@ -1178,10 +1178,10 @@ defmodule Enum do
   end
 
   @doc """
-  Returns the biggest of the elements in the enumerable according
+  Returns the maximal element of the elements in the enumerable according
```

Comment:
This sounds really weird to me 😕 What if we go with "Returns the maximal element in the enumerable"?